app/uk/gov/hmrc/traderservices/controllers/FormFieldMappings.scala

```diff
@@ -224,6 +239,7 @@ object FormFieldMappings {
       first(
         constraint[String]("contactEmail", "required", _.nonEmpty),
         Constraints.emailAddress(errorMessage = "error.contactEmail"),
```

Comment:
The same regex is used for import as well as export. May be extract to a variable so that it can be used elsewhere if needed in future?